src/core_plugins/kibana/public/dashboard/dashboard_state_manager.js

```diff
@@ -128,27 +126,8 @@ export class DashboardStateManager {
     });
   }
 
-  /**
-   * For each embeddable config in appState that differs from that in the redux store, trigger the change listeners
-   * using the appState version as the "source of truth". This is because currently the only way to update an embeddable
-   * config from the dashboard side is via the url. Eventually we want to let users modify it via a "reset link" in
-   * the panel config, or even a way to modify it in the panel config. When this is introduced it would go through
-   * redux and we'd have to update appState. At that point, we'll need to handle changes coming from both directions.
-   * Ideally we can introduce react-redux-router for a more seamless way to keep url changes and ui changes in sync.
-   * ... until then... we are stuck with this manual crap. :(
-   * Fixes https://github.com/elastic/kibana/issues/15720
-   */
-  triggerEmbeddableConfigUpdateListeners() {
```

Comment:
These changes now get sent to the embeddable via an onContainerStateChanged call when the EmbeddableViewport detects a change in the return value of getContainerState.